src/openrct2/title/TitleScreen.cpp

```diff
@@ -283,6 +287,11 @@ extern "C"
         }
     }
 
+    uint16 title_get_config_sequence()
```

Comment:
Why is it returning `uint16`?

Reply:
It was returning uint16 since that was what `title_get_current_sequence()` (The function below) returned. I'll change both of them to size_t in the title editor refactor PR.